apps/api/tests/test_admin.py

```diff
@@ -128,6 +140,37 @@ def test_no_decision_from_no_reviews() -> None:
     assert record["decision"] is None
 
 
+@patch("services.mongodb_handler.update", autospec=True)
+@patch("services.mongodb_handler.retrieve_one", autospec=True)
+@patch("services.mongodb_handler.retrieve", autospec=True)
+def test_confirm_attendance_when_status_confirmed(
+    mock_mongodb_handler_retrieve: AsyncMock,
+    mock_mongodb_handler_retrieve_one: AsyncMock,
+    mock_mognodb_handler_update: AsyncMock,
+) -> None:
+    """Test that confirmed status changes to attending."""
+
+    mock_mongodb_handler_retrieve_one.return_value = DIRECTOR_IDENTITY
+    mock_mongodb_handler_retrieve.return_value = [
+        {
+            "_id": "edu.uc.tester",
+            "role": "applicant",
+            "status": Status.CONFIRMED,
+        }
+    ]
```

Comment:
Chore: Could we also include the two other conversion situations in the test data?

Reply:
Instead of repeating the same test three times, we could include three applicants of different statuses in the mocked return value, also to simulate a more realistic scenario.